tests/integration-tests/configs/develop.yaml

```diff
@@ -13,6 +13,18 @@ test-suites:
           instances: ["c5n.18xlarge"]
           oss: ["rocky8"]
           schedulers: ["slurm"]
+        - regions: [ "euw1-az1" ]  # do not move, unless capacity reservation is moved as well
```

Comment:
Do we want these tests to run daily or weekly?

Also, if its not necessary to run them with the same instance type we can leverage the already existing tests with different instances like p4d, c6gn.16xlarge or hpc6id that are running with alinux2.


However, if c5n.18xlarge are the only instances that we need to run these tests daily with, then I suggest using 
```
oss: {{ common.OSS_COMMERCIAL_X86 }}

```
instead of using multiple sections.